src/main/java/hunternif/mc/atlas/client/BiomeTextureConfig.java

```diff
@@ -76,7 +110,8 @@ protected void saveData(JsonObject json, BiomeTextureMap data) {
 			// Only save biomes 0-256 in this config.
 			// The rest goes into ExtTileTextureConfig
 			if (biomeID >= 0 && biomeID < 256) {
-				json.addProperty(String.valueOf(biomeID), data.textureMap.get(biomeID).name);
+				String key = Biome.REGISTRY.getObjectById(biomeID).getRegistryName().toString();

```

Comment:
```java
String key = ((ForgeRegistry<Biome>) ForgeRegistries.BIOMES).getValue(biomeID);
```

Use the Forge provided classes rather than Vanilla ones.